### ISSUE TITLE
epub and mobi (Kindle) support

### ISSUE THREAD

**Issue by TomasLinhart** [runpaint/read-ruby#46]
I would love to read this on my Kindle. Would be possible to have mobi version?

Thanks!

**Comment by runpaint** [comment]
Yes, ebook formats are definitely on the cards. Unfortunately, I can't give a time-frame.
